spec/linkedin-scraper/profile_spec.rb

```diff
@@ -103,7 +103,7 @@
   end
 
   describe '#organizations' do
-    it 'returns an array of organization hashes for the profile' do
+    pending 'returns an array of organization hashes for the profile' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.